atoMEC/models.py

```diff
@@ -383,3 +383,98 @@ def CalcEnergy(
         }
 
         return output_dict
+
+    def CalcPressure(
+        self,
+        atom,
+        nmax,
```

Comment:
I suggest using a default value here, `nmax=config.nmax` since this was set earlier in `CalcEnergy`.